integration/class-enable-mastodon-apps.php

```diff
@@ -233,4 +249,104 @@ public static function api_search( $search_data, $request ) {
 
 		return $search_data;
 	}
+
+	public function api_get_posts_query_args( $args ) {
+		if ( isset( $args['author'] ) && is_string( $args['author'] ) ) {
+			$uri = Webfinger_Util::resolve( $args['author'] );
+			if ( $uri && ! is_wp_error( $uri ) ) {
+				$args['activitypub'] = $uri;
+				unset( $args['author'] );
+			}
+		}
+
+		return $args;
+	}
+
+	public static function api_statuses_external( $statuses, $args ) {
+		if ( ! isset( $args['activitypub'] ) ) {
+			return $statuses;
+		}
+
+		$data = get_remote_metadata_by_actor( $args['activitypub'] );
+
+		if ( ! $data || is_wp_error( $data ) || ! isset( $data['outbox'] ) ) {
+			return $statuses;
+		}
+
+		$response = \Activitypub\Http::get( $data['outbox'], true );
```

Comment:
Can you add that as `use Activitypub\Http` to the head, to not have namespaces in the code?